.github/workflows/docs-guide-test.yml

```diff
@@ -1,5 +1,10 @@
 name: Test Docs Guide
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+      - '1.*'
+  pull_request:
```

Comment:
This change is so that we only build on release branches and `main`. Otherwise, if you push a branch to open a PR, we would run the same CI jobs twice.